CONTRIBUTING.md

```diff
@@ -23,3 +23,16 @@ an issue](https://github.com/MaterializeInc/materialize/issues/new/choose). As
 Materialize is under rapid development, it is helpful if you report the version
 of Materialize that you are using, and if it a crash report, the stack trace
 printed from the `materialized` binary.
+
+### Landing PRs + communicating changes
+
+When landing large or substantial changes, we want to make sure users are aware of the work you're doing! This means we require:
+
+- Coordination with a technical writer to generate or update user-facing documentation that will show up on <materialize.io/docs>. If you have questions, open an issue with the `A-docs` tag.
```

Comment:
Should we mention who the technical writer is here?

Reply:
I don't like hardcoding organizational structure into the docs, e.g. I could get run over by a bus and y'all would need to think to grep for me to replace me. I think that creating tags or just asking someone is fine given the low frequency with which I expect this to happen.